test/extended/router/stress.go

```diff
@@ -71,7 +71,9 @@ var _ = g.Describe("[sig-network][Feature:Router][apigroup:route.openshift.io]",
 				Kind: "ClusterRole",
 				Name: "system:router",
 			},
-		}, metav1.CreateOptions{})
+		}
+
+		_, err = oc.AdminKubeClient().RbacV1().RoleBindings(ns).Create(context.Background(), roleBinding, metav1.CreateOptions{})
```

Comment:
Why this change when `roleBinding` is used only once?

Reply:
Leftover from previous change.